geomstats/information_geometry/multivariate_normal.py

```diff
@@ -225,6 +351,125 @@ def pdf(x):
         return pdf
 
 
+class MultivariateGeneralNormalDistributions(InformationManifoldMixin, ProductManifold):
+    """Class for the manifold of multivariate normal distributions.
+
+    This is the class for multivariate normal distributions on the $n$-dimensional
+    Euclidean space. Each distribution is represented by the concatenation of its
+    mean vector and its covariance matrix reshaped in a $n^2$-vector.
+
+    Parameters
+    ----------
+    n : int
+        Dimension of the sample space of the multivariate normal distribution.
+    """
+
+    def __init__(self, n):
+        super().__init__(factors=[Euclidean(n), SPDMatrices(n)])
+        self.n = n
+
+    def reformat(self, point):
+        """Reformat point."""
```

Comment:
+ Missing docstring for parameter point.
+ Missing explanation on what "reformat" means.

how about `from_array_to_tuple` or `unconcatenate` or `unpack` or something like this?